src/data/reports/patientReport.ts

```diff
@@ -0,0 +1,5 @@
+export interface ReportPatient{
+    name: string;
+    lastname: string;
```

Comment:
what is the difference between ReportPatient y Reports? looks like you have the same elements you can remove one of these classes

Reply:
I don't know why it's still visible but I already deleted it.

Reply:
If it's visible it means that you didn't update correctly your branch